security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.cloud.securitycenter.v2.BigQueryExport;
+import com.google.cloud.securitycenter.v2.BigQueryExportName;
+import com.google.cloud.securitycenter.v2.CreateBigQueryExportRequest;
+import com.google.cloud.securitycenter.v2.DeleteBigQueryExportRequest;
+import com.google.cloud.securitycenter.v2.GetBigQueryExportRequest;
+import com.google.cloud.securitycenter.v2.ListBigQueryExportsRequest;
+import com.google.cloud.securitycenter.v2.OrganizationLocationName;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient.ListBigQueryExportsPagedResponse;
+import com.google.cloud.securitycenter.v2.UpdateBigQueryExportRequest;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import vtwo.bigquery.CreateBigQueryExport;
+import vtwo.bigquery.DeleteBigQueryExport;
+import vtwo.bigquery.GetBigQueryExport;
+import vtwo.bigquery.ListBigQueryExports;
+import vtwo.bigquery.UpdateBigQueryExport;
+
+public class BigQueryExportIT {
+  public static final String ORGANIZATION_ID = "test-organization-id";
+  public static final String PROJECT_ID = "test-project-id";
+  private static final String LOCATION = "global";
+  private static final String BQ_DATASET_NAME = "test-dataset-id";
+  private static final String BQ_EXPORT_ID = "test-export-id";
+  private static MockedStatic<SecurityCenterClient> clientMock;
+
+  @BeforeClass
+  public static void setUp() {
+    clientMock = Mockito.mockStatic(SecurityCenterClient.class);
+  }
+
+  @AfterClass
+  public static void cleanUp() {
+    clientMock.reset();
+  }
+
+  @Test
+  public void testCreateBigQueryExport() throws IOException {
+    // Mock SecurityCenterClient.
+    SecurityCenterClient client = mock(SecurityCenterClient.class);
+    clientMock.when(SecurityCenterClient::create).thenReturn(client);
```

Comment:
Can you also move these two lines to the setUp() method? (same for other methods)